luigi/db_task_history.py

```diff
@@ -19,6 +19,21 @@
 Provides a database backend to the central scheduler. This lets you see historical runs.
 See :ref:`TaskHistory` for information about how to turn out the task history feature.
 """
+#
+# Description: Added codes for visualization of how long each task takes
+# running-time until it reaches the next status (failed or done)
+# At "{base_url}/tasklist", all completed(failed or done) tasks are shown.
+# At "{base_url}/tasklist", a user can select one specific task to see
+# how its running-time has changed over time.
+# At "{base_url}/tasklist/{task_name}", it visualizes a multi-bar graph
+# that represents the changes of the running-time for a selected task
+# up to the next status (failed or done).
+# This visualization let us know how the running-time of the specific task
+# has changed over time.
+#
+# Copyright 2015 Naver Corp.
```

Comment:
Oh, I see. Please let me know what to do then. :) 
It is required for me to put the copyright part "# Copyright 2015 Naver Corp. # Author Yeseul Park (yeseul.park@navercorp.com)" in the header for every files that I've modified. Moreover, I should write down the # Description part at least once.